modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/TeamMemberInputs/index.jsx

```diff
@@ -97,9 +133,7 @@ const TeamMemberInputs = ({
 						placeholder={placeholderEmail}
 						required
 						type="email"
-						validations={[
-							(value) => isValidEmail(value, bannedDomains),
-						]}
+						validations={[() => validateEmail]}
```

Comment:
This is working?

Syntax is weird, instead of using the validateEmail as value, can it be a function and replace the `useMemo` by `useCallback`?

```suggestion
						validations={[() => validateEmail()]}
```

Reply:
```suggestion
						validations={validateEmail}
```